src/templates/layouts/healthCareLocalFacility/index.tsx

```diff
@@ -109,7 +110,7 @@ export function HealthCareLocalFacility({
             >
               Location and contact information
             </h2>
-            <div className="region-list usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row facility vads-u-margin-bottom--4">
+            <div className="region-list usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column mobile-lg:vads-u-flex-direction--row facility vads-u-margin-bottom--2p5 vads-u-margin-bottom--4">
```

Comment:
Just re-copied this from `content-build` because it wasn't displaying to the side like I expected.